kustomize/commands/localize/localize.go

```diff
@@ -74,6 +110,12 @@ kustomize localize https://github.com/kubernetes-sigs/kustomize//api/krusty/test
 Cannot specify for remote targets, as scope is by default the containing repo.
 If not specified for local target, scope defaults to target.
 `)
+	cmd.Flags().BoolVar(&f.noVerify,
+		"no-verify",
+		false,
+		`Does not verify that the outputs of kustomize build for target and newDir are the same after localization.
+		If not specified, noVerify defaults to false and will run kustomize build.
```

Comment:
WDYT of printing "this flag defaults to false" instead of "noVerify defaults to false" here?